faker/providers/passport/en_US/__init__.py

```diff
@@ -0,0 +1,112 @@
+import random
+
+from datetime import date, timedelta
+
+from .. import Provider as PassportProvider
+
+
+class Provider(PassportProvider):
+    """Implement passport provider for ``en_US`` locale.
+
+    Sources:
+
+    - https://travel.state.gov/content/travel/en/passports/passport-help/next-generation-passport.html
+    - https://www.vitalrecordsonline.com/glossary/passport-book-number
+    """
+
+    DEFAULT_DAY = date.today()
```

Comment:
I'm afraid this would only get evaluated at declaration time, and not at runtime, resulting in the wrong value. Eg imagine the typical webserver scenario, where the process starts and lasts for days. The `DEFAULT_DAY` would be stuck at the day the webserver was launched.